### PR TITLE
[SAIC-351] Port config options for MySQL and RabbitMQ

### PR DESCRIPTION
**Pull request by miarmak**
Added port option to the MySQL and RabbitMQ config sections for setting
up custom ports.

### CODE REVIEW

**Comment by realsystem** [comment]
+1

File: cloudferrylib/os/identity/keystone.py [hunk 1]
```diff
@@ -480,9 +480,13 @@ def _render_template(self, name_file, args):
     def check_rabbitmq(self):
         credentials = pika.PlainCredentials(self.config.rabbit.user,
                                             self.config.rabbit.password)
-        for host in self.config.rabbit.hosts.split(","):
-            pika.BlockingConnection(pika.ConnectionParameters(
-                host=host.strip(), credentials=credentials))
+
+        for host_with_port in self.config.rabbit.hosts.split(","):
+            host, port = host_with_port.split(':')
```
Comment (roman-verchikov) [inline cloudferrylib/os/identity/keystone.py]: Will host *always* have port?.. This code will fail with `TypeError` if not.

Comment (miarmak) [inline cloudferrylib/os/identity/keystone.py]: I guess yes, they will. At least all OpenStack services define RabbitMQ hosts in this manner (with `host:port` notation):
http://docs.openstack.org/high-availability-guide/content/_configure_openstack_services_to_use_rabbitmq.html

**Comment by roman-verchikov** [comment]
should be good then, +1

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**